content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -790,7 +790,7 @@ Get the Pod's container:
 kubectl get pod web-2 --template '{{range $i, $c := .spec.containers}}{{$c.image}}{{end}}'
 ```
 ```
-registry.k8s.io/nginx-slim:0.7
+registry.k8s.io/nginx-slim:0.24
```

Comment:
I'm not sure this works. We need two **different** image tags; they can be any two tags, but they can't be the same as each other.